File: template/src/main/java/ru/surfstudio/standard/interactor/common/error/ConversionException.java
package ru.surfstudio.standard.interactor.common.error;

/**
 * ошибка парсинга ответа
 */
public class ConversionException extends NetworkException {
    public ConversionException(String message, Throwable cause) {
        super(message, cause);
    }

    public ConversionException(String message) {
        super(message);
    }
}

File: template/src/main/java/ru/surfstudio/standard/interactor/common/error/ConversionException.kt
package ru.surfstudio.standard.interactor.common.error

/**
 * ошибка парсинга ответа
 */
class ConversionException : NetworkException {
    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(message: String) : super(message) {}
}
